ocean_lib/models/test/test_erc721_token.py

```diff
@@ -384,6 +384,45 @@ def test_success_update_metadata(web3, config, publisher_wallet, consumer_wallet
     assert metadata_info[3] is True
     assert metadata_info[0] == "http://foourl"
 
+    # Update tokenURI and set metadata in one call
+    tx = erc721_token.set_metadata_token_uri(
+        metadata_state=1,
+        metadata_decryptor_url="http://foourl",
+        metadata_decryptor_address="0x123",
+        flags=web3.toBytes(hexstr=BLOB),
```

Comment:
this is not a hexstr, do we test with BLOB or "BLOB"?

Reply:
BLOB is a constant in web3_internal/constants.py. I declared for testing purpose (from contracts)
![Screenshot from 2022-01-20 02-52-39](https://user-images.githubusercontent.com/50501033/150325227-7592dfb0-2fab-43a8-b350-f1aa8f43df11.png)